test/__mocks__/due.diligence.mock.ts

```diff
@@ -2,10 +2,16 @@ import { DueDiligence } from "../../src/model/due.diligence.model";
 import {
   ADDRESS,
   IDENTITY_ADDRESS_REQ_BODY_EMPTY_MOCK,
+  IDENTITY_ADDRESS_REQ_BODY_MAX_LENGTH_MOCK,
   IDENTITY_ADDRESS_REQ_BODY_MOCK,
 } from "./fields/address.mock";
 import { DATE, EMPTY_DATE } from "./fields/date.mock";
 
+const TEN_CHARACTERS_LENGTH = "LKJHG.asdf";
+const FIFTY_CHARACTERS_LENGTH = "ABCDEabcde0123456789QWERTYUIOPqwertyuiopZXCVBzxcvb";
+
+const MAX_256 = FIFTY_CHARACTERS_LENGTH.repeat(5) + TEN_CHARACTERS_LENGTH;
```

Comment:
This constant name is confusing - it contains a string that's 260 characters in length.

Reply:
Updated it to now be 256 characters long and adding an extra character for the test